projects/laji/src/app/shared-modules/select/select/select.component.html

```diff
@@ -1,28 +1,38 @@
 <div class="lj-select" [ngClass]="{'lj-close': open, 'lj-open': !open, 'lj-disabled': disabled}">
-  <strong *ngIf="title !== undefined && title !== ''" class="lj-title link" (click)="toggle($event, filter)">{{ title }}<laji-info *ngIf="info" [html]="info"></laji-info></strong>
+  <strong *ngIf="title !== undefined && title !== '' && subCategories.length === 0" class="lj-title link" (click)="toggle($event, filter)">{{ title }}<laji-info *ngIf="info" [html]="info"></laji-info></strong>
   <ng-container *ngIf="multiple; else select">
     <ng-container *ngIf="!useFilter; else multiWithFilter">
       <ng-container *ngIf="!open">
         <span class="item" *ngFor="let option of selectedOptions; trackBy: track">
-          <label class="lj-item selected" (click)="labelClick($event)">
-            <laji-checkbox [value]="true" (valueChange)="remove(option.id)"></laji-checkbox> {{ option.value}}
+          <label class="lj-item selected" [ngClass]="{'subCategoriesOptions': subCategories.length > 0, '': subCategories.length === 0}" (click)="labelClick($event)">
+            <ng-container *ngIf="subCategories.length === 0 && !isParentFilter else checkboxSubCategoriesGlobal">
+              <laji-checkbox [value]="true" (valueChange)="remove(option.id)"></laji-checkbox> {{ option.value}}
+            </ng-container>
+            <ng-template #checkboxSubCategoriesGlobal>
+              <laji-checkbox [threeState]="isParentFilter" [hasSubCategories]="true" (valueChange)="remove(option.id)" [value]="checkStatus(option.id, (isParentFilter ? 'GLOBAL': subCategory))"></laji-checkbox>{{ option.value }}
```

Comment:
having a function as input value is not optimal since it's triggered on each change detection cycle and the checkStatus seems to have quite a lot of logic in it. Could this logic of what is actually selected moved higher up to select-subcategories.component?